2013/lar-syntactical.json

```diff
@@ -17,7 +17,7 @@
             "property": "actionDate",
             "condition": "call",
             "function": "isActionDateInActivityYear",
-            "args": ["actionDate", "transmittalSheet.activityYear"]
+            "args": ["actionDate", "_HMDA_JSON.hmdaJson.hmdaFile.transmittalSheet.activityYear"]
```

Comment:
I'll refactor the rest of the code to add hmdaFile itself to the context list so that this line can be reverted back.